### PR TITLE
"Select filter" button is active without selecting any saved filter w…

### PR DESCRIPTION
**Pull request by winnie-chiu**
…ithin Topic/Messages/Saved filters

- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

1. Only enable "Select filter" button when there is a saved filter selected.
2. Reset the selected saved filter when any saved filter is deleted.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by kristi-dev — APPROVED** [review]

**Review by Mgrdich — COMMENTED** [review]
can we write a minor test indicating that the button is disabled after deleting the Filters

File: kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/__tests__/SavedFilters.spec.tsx [hunk 1]
```diff
@@ -158,6 +163,11 @@ describe('SavedFilter Component', () => {
       await userEvent.click(screen.getByRole('button', { name: 'Confirm' }));
       expect(deleteMock).toHaveBeenCalledTimes(1);
       expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
+
+      const selectFilterBtn = screen.getByRole('button', {
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/__tests__/SavedFilters.spec.tsx:167]: i think here the query have to be more specific cause it it finding many and that is why it is failing the tests

**Comment by Haarolean** [comment]
@BulatKha take a look please

**Review by BulatKha — APPROVED** [review]
Works ok with current messaging.
Messages V2 check not possible yet.
Approving the PR.

**Pull request merged**